MingoDB/tests/Connect.js

```
@@ -0,0 +1,42 @@
+const MingoDB = require('../src/MingoDB')
+const fs = require('fs')
+const expect = require('chai').expect
+
+describe('MingoDB - #connect()', function() {
+  const DBDirectoryPath = 'C:/Users/Anat/Desktop/Documents/training/NodeJsTraining/MingoDB/tests/DBTestDirectory'
+  const testFilePath = 'C:/Users/Anat/Desktop/Documents/training/NodeJsTraining/MingoDB/tests/testFile.txt'
+
+  before(function(done) {
+    fs.mkdir(DBDirectoryPath, function(){
+      fs.open(testFilePath, 'w', done)
+    })
+  })
+    
+  after(function(done) {
+    fs.rmdir(DBDirectoryPath, function() {
+      fs.unlink(testFilePath, done)
+    })
+  })
+  
+  afterEach(function() {
+    fs.chmodSync(DBDirectoryPath, 0o777)
+  })
+
+  it('creates a new db object', function(done) {
```

When you describe tests better write what is the test purpose "Should create a new db object", "Should throw an error when the dbFolderPath isn't string.."